### ISSUE TITLE
Availability labels lack sufficient color contrast

### ISSUE THREAD

**Issue by axamei** [pulibrary/orangelight#1060]
Level AA violation: Elements must have sufficient color contrast. 

Currently, background colors for labels are too light for the white text. 

<img width="221" alt="screen shot 2017-12-18 at 11 01 47 am" src="https://user-images.githubusercontent.com/8161144/34115204-eb3bbb68-e3e2-11e7-8830-0a09763e3e41.png">
<img width="111" alt="screen shot 2017-12-18 at 11 01 37 am" src="https://user-images.githubusercontent.com/8161144/34115205-eb48874e-e3e2-11e7-8cfd-a363874632a5.png">
<img width="110" alt="screen shot 2017-12-18 at 11 01 31 am" src="https://user-images.githubusercontent.com/8161144/34115206-eb53ac32-e3e2-11e7-8869-9a65ba88a2c4.png">

Darken background or adjust display of availability information. 

See: [availability.scss](https://github.com/pulibrary/orangelight/blob/master/app/assets/stylesheets/components/availability.scss#L18-L31) (Also need to fix: Color values should be pulled from variables. These are coming from [Bootstrap](https://github.com/twbs/bootstrap-sass/blob/5d6b2ebba0c2a5885ce2f0e01e9218db3d3b5e47/assets/stylesheets/bootstrap/_labels.scss#L41-L66).)
